Makefile

```diff
@@ -142,7 +142,8 @@ license-header-check:
 	           exit 1; \
 	   fi
 
-.PHONY: fixture-nethttp fixture-gin fixture-databasesql
+.PHONY: fixture-nethttp fixture-gin fixture-databasesql fixture-nethttp-custom
```

Comment:
If adding a fixture here, it is worth adding also one in the github workflows in `kind.yml`